en/docs/develop-components/develop-integrations/develop-a-manual-task.md

```diff
@@ -1,106 +1,97 @@
-# Develop a Manual Task
+# Develop a manual task
 
-This guide walks you through the steps to develop, deploy, test, and observe a manual task using Choreo.  
+This guide walks you through the steps to develop, deploy, test, and observe a manual task using Choreo.
 
 In this guide, you will:
 
-  - Develop a manual task to fetch the weather forecast for a specified location for the next 24 hours from the  [OpenWeatherMap](https://openweathermap.org) API.
-  - Process the weather data to a specific format.
-  - Send the formatted data to a specified email address.
+- Develop a manual task to fetch the weather forecast for a specified location for the next 24 hours from the [OpenWeatherMap](https://openweathermap.org) API.
+- Process the weather data into a specific format.
+- Send the formatted data to a specified email address.
 
 ## Prerequisites
 
-Before you try out the steps in this guide, complete the following:
+Before you begin:
 
- - If you are signing in to the Choreo Console for the first time, create an organization as follows:
-    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in.
-    2. Enter a unique organization name. For example, `Stark Industries`.
+1. **Create a Choreo Organization**:
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. See https://developers.google.com/style/text-formatting and https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines.